README.md

```diff
@@ -11,6 +11,11 @@ You can run your application in dev mode that enables live coding using:
 ./mvnw quarkus:dev
 ```
 
+## Build Docker image
+```
+docker build -f docker/Dockerfile.jvm -t t-learn/backend .
```

Comment:
should be `./mvnw clean package -Dquarkus.container-image.build=true`

Reply:
For this I also removed the commit that moved the Docker directory, since Quarkus per default needs them to be in `/src/main`